spec/lib/scholrax/importer/handle_file_reader_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+module Scholrax::Importer
+  RSpec.describe HandleFileReader do
+
+    describe "call" do
+      let(:handle_file_path) { Rails.root.join('spec', 'fixtures', 'dspace_export', 'sample', 'handle') }
+
+      subject { described_class.new(handle_file_path) }
+      it "returns the expected handle" do
+        expect(subject.call).to eq ('12345/98765')
```

Comment:
@ad96 Because of the change in method name that you made in HandleFileReader, that class no longer has a `call` method.  You need to change the test accordingly.